segmentifyandroidsdk/src/main/java/com/segmentify/segmentifyandroidsdk/utils/Constant.kt

```diff
@@ -37,6 +38,7 @@ class Constant {
         val bannerImpressionStep = "impression"
         val bannerClickStep = "click"
         val bannerUpdateStep = "update"
+        val searchStep = "search"
```

Comment:
what is the use of search step?

Reply:
fun sendSearchClickView use this constant

Reply:
@ebruersoy I saw that you used that variable to set **interactionModel.type**. Do we really use **search** type in interactions?

We have several types (such as impression, widget-view, click) but I'm not sure about search. What is the use of search interaction type?

Reply:
@mesutcank we send a clickView event when a product is clicked from **search panel** for reporting the search data. Parameters are like this:
instanceId: "product" (because it is clicked from product side of search panel, if it is clicked from sidebar like category or brand, instanceId would be "catgeory" or "brand") 
type: "search" (to understand this click is from search, not from reco campaign, so this report data is written to the search_logs table, not to campaign_logs table.)

Reply:
@ebruersoy so the instanceId is only "product" for SDK case, right?

Reply:
and SEARCH also